libnavigator/src/main/java/com/mapbox/navigation/navigator/SensorMapper.kt

```diff
@@ -0,0 +1,57 @@
+package com.mapbox.navigation.navigator
+
+import android.hardware.Sensor
+import android.hardware.SensorEvent
+import android.os.Build
+import android.util.Log
+import com.mapbox.navigator.NavigatorSensorData
+import com.mapbox.navigator.SensorType
+import java.util.Date
+
+object SensorMapper {
```

Comment:
Capturing from internal discussion with @kmadsen that `SensorMapper` could live outside of the SDK as it's only needed for writing data to files. That'd avoid the need of having to update it if it needs to be extended (new sensors are available) which means no need new releases in the future to support new data and less to maintain on our side.